core/src/main/scala/cats/SemigroupK.scala

```diff
@@ -31,9 +31,9 @@ import simulacrum.{op, typeclass}
   /**
    * Compose two SemigroupK intsances.
    */
-  def composedWith[G[_]: SemigroupK]: SemigroupK[λ[α => F[G[α]]]] =
+  def compose[G[_]: SemigroupK]: SemigroupK[λ[α => F[G[α]]]] =
```

Comment:
I think the right thing to do here is to add a `compose` to `Alternative` which composes a pair of `Alternatives` to an `Alternative`. I can add that to this PR or follow up afterwards.